Monika After Story/game/event-handler.rpy

```diff
@@ -1425,6 +1426,9 @@ init -1 python in evhand:
         "unlock_prompt",
     ]
 
+    # Used to catch "idle_exp: EXP, DURATION" or "idle_exp: TAG"
+    RET_KEY_PATTERN_IDLE_EXP = re.compile(r"(?:(?<=\|)|(?<=^))\s*idle_exp\s*:\s*(?:(?P<exp>\d[a-z]{3,13})\s*,\s*(?P<duration>\d)|(?P<tag>\w+))\s*(?:(?=\|)|(?=$))")
```

Comment:
While it's unlikely we'll need to use something over 10 seconds. We should still account for it.
```suggestion
    RET_KEY_PATTERN_IDLE_EXP = re.compile(r"(?:(?<=\|)|(?<=^))\s*idle_exp\s*:\s*(?:(?P<exp>\d[a-z]{3,13})\s*,\s*(?P<duration>\d+)|(?P<tag>\w+))\s*(?:(?=\|)|(?=$))")
```